x/fees/keeper/fees.go

```diff
@@ -140,48 +140,52 @@ func (k Keeper) IsFeeRegistered(
 	return store.Has(contract.Bytes())
 }
 
-// GetFeesInverseRaw returns all contracts registered by a deployer as
-// types.DevFeeInfosPerDeployer
-func (k Keeper) GetFeesInverseRaw(ctx sdk.Context, deployerAddress sdk.AccAddress) (types.DevFeeInfosPerDeployer, bool) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixInverse)
-	bz := store.Get(deployerAddress.Bytes())
-	if len(bz) == 0 {
-		return types.DevFeeInfosPerDeployer{}, false
-	}
-	var addressList types.DevFeeInfosPerDeployer
-	k.cdc.MustUnmarshal(bz, &addressList)
-	return addressList, true
-}
-
 // GetFeesInverse returns all contracts registered by a deployer as []common.Address
 func (k Keeper) GetFeesInverse(ctx sdk.Context, deployerAddress sdk.AccAddress) []common.Address {
```

Comment:
```suggestion
func (k Keeper) GetDeployerFees(ctx sdk.Context, deployerAddress sdk.AccAddress) []common.Address {
```

The term "inverse" is not clear, we have a similar one-to-many relationship for gasMeters and incentives.